packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -262,28 +261,31 @@ export class EVMAAWallet<B extends EVMBlockchainIncludingTestnet = EVMBlockchain
             });
 
             const smartAccountSigner = walletClientToSmartAccountSigner(walletClientSigner);
-            const sessionKeyValidator = await signerToSessionKeyValidator(this.publicClient, {
+            const sessionKeySigner = toECDSASigner({
                 signer: smartAccountSigner,
-                validatorData: {
-                    paymaster: oneAddress,
-                },
-            });
+              })
 
+            const sessionKeyValidator = await toPermissionValidator(this.publicClient, {
+                entryPoint: ENTRYPOINT_ADDRESS_V07,
+                signer: sessionKeySigner,
+                policies: [],
```

Comment:
I'm curious about leaving policies as an empty array. 

From my understanding, we want to get full control through the session key right? If so, I understand the way to go here as the [sudo policy ](https://docs.zerodev.app/sdk/permissions/policies/sudo). 

WDYT @lluiserena ?

Reply:
I don't think we want sudo policy because we cannot have access to transfer fungible tokens from crossmint.com
but that does bring up the question–does this empty policy allow that still? Or do we need to add policy based on 721/1155 functions?